src/c++/perf_analyzer/main.cc

```diff
@@ -589,19 +604,78 @@ Usage(char** argv, const std::string& msg = std::string())
             << std::endl;
   std::cerr << std::endl;
   std::cerr << "III. SERVER DETAILS: " << std::endl;
-  std::cerr << std::setw(9) << std::left << " -u: "
+  std::cerr << std::setw(38) << std::left << " -u: "
             << FormatMessage(
                    "Specify URL to the server. When using triton default is "
                    "\"localhost:8000\" if using HTTP and \"localhost:8001\" "
                    "if using gRPC. When using tfserving default is "
                    "\"localhost:8500\". ",
-                   9)
+                   38)
             << std::endl;
-  std::cerr << std::setw(9) << std::left << " -i: "
+  std::cerr << std::setw(38) << std::left << " -i: "
             << FormatMessage(
                    "The communication protocol to use. The available protocols "
                    "are gRPC and HTTP. Default is HTTP.",
-                   9)
+                   38)
+            << std::endl;
+  std::cerr << std::setw(38) << std::left
+            << " --ssl-grpc-root-certifications-file: "
+            << FormatMessage(
+                   "Path to file containing the PEM encoding of the server "
```

Comment:
are there defaults for any of these? Is there a master option that enables all of the other options?

Reply:
> are there defaults for any of these?

Yes, their defaults are here: https://github.com/triton-inference-server/client/blob/matthewkotila-pa-ssl-options/src/c%2B%2B/perf_analyzer/main.cc#L798-L810

> Is there a master option that enables all of the other options?

What do you mean by this?